Dockerfile

```diff
@@ -10,11 +10,14 @@ RUN conda config --add channels defaults && \
     conda config --add channels conda-forge && \
     conda config --set auto_update_conda False
 RUN : 'Make sure we get the (working) conda we want before installing the rest.' && \
-    sed -nE \
-        -e 's/\s*#.*$//' \
-        -e 's/^(conda([><!=~ ].+)?)$/\1/p' \
-        /tmp/repo/bioconda_utils/bioconda_utils-requirements.txt \
-        | xargs -r conda install -y
+    : 'FIXME: temporary workaround: install conda=4.8.2 due to solver stalls with conda=4.6.14' && \
+    : '       (actual conda version gets still installed via bioconda_utils-requirements.txt!)' && \
+    conda install -y conda=4.8.2 && \
```

Comment:
I used `4.8.2` here just because it's the same as the one as in https://github.com/bioconda/bioconda-common/pull/14 .